src/components/ProductCard.test.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
remove this comment :)

Reply:
Same here - I opted to comment it out instead of refactoring your code.